haskell-overlays/text-jsstring/default.nix

```diff
@@ -60,5 +60,5 @@ self: super: {
       self.ghcjs-base
     ];
   });
-  aeson = self.callPackage self._dep.aeson {};
+  aeson = dontCheck (self.callCabal2nix "aeson" self._dep.aeson {});
```

Comment:
We should really try to keep these tests enabled given the hacky nature of text-jsstring patches on Aeson.